LEAF_Request_Portal/templates/view_reports.tpl

```diff
@@ -688,6 +693,72 @@ function showJSONendpoint() {
 	dialog_message.show();
 }
 
+/**
+ * Purpose: Update Request Titles
+*/
+
+ function changeTitle(form_data) {
+     console.log(form_data);
+	dialog.setContent('Title: <input type="text" id="title" style="width: 300px" name="title" value="<!--{$title|escape:'quotes'}-->" /><input type="hidden" id="CSRFToken" name="CSRFToken" value="<!--{$CSRFToken}-->" />');
```

Comment:
Please make the id and name more unique.  This will be important as things are moved more Dom control oriented.

Reply:
changed them to reportTitle and updated references

Reply:
edit: changed it again to recordTitle because reportTitle collided with one of the existing css styles.